src/pages/guide/language/index.md

```diff
@@ -29,26 +29,24 @@ How Reason Works
 =========================
 
 The OCaml compiler is organized into several stages, which are exposed as
-libraries. `Reason` replaces part of the compiler toolchain with a completely
+libraries. Reason replaces part of the compiler toolchain with a completely
 new syntax parser that is more approachable, while still fully compatible with
-the rest of the compiler.  `Reason` also implements a new source printer which
+the rest of the compiler.  Reason also implements a new source printer which
 integrates into your IDE and the new custom [`REPL`](./tools.html#repl).
 
 ###### Why OCaml?
 
-OCaml is a great tool for writing highly expressive, functional
+OCaml is a great language for writing highly expressive, functional
 *or* imperative code, with type inference and fast runtime performance.
 Because of these properties, OCaml has helped
 Facebook quickly build scalable infrastructure such as
 [Hack](http://hacklang.org/), [Flow](http://flowtype.org/), and
 [Infer](http://fbinfer.com/). It is also used for other performance sensitive
-applications in the financial industry (Jane Street, Bloomberg). At the same
-time, OCaml has a very mature (*and still growing*) ecosystem for targeting
-browser and `JavaScript` environments with a focus on language interoperability
-and integration with existing `JavaScript` code.
-
-
-`Reason`'s non-invasive approach to the `OCaml` compiler allows `Reason` code
-to take advantage of all of the existing `OCaml` compiler
-optimizations/backends such as bare metal `ARM`, `x86`, and even `JavaScript`
-compilation.
+applications in the financial industry (Jane Street, Bloomberg).
+
+Reason's non-invasive approach to the OCaml compiler allows Reason code
+to take advantage of all of the existing OCaml compiler
+optimizations/backends such as bare metal ARM, x86, and even JavaScript
+compilation. OCaml has a very mature (*and still growing*) ecosystem for targeting
+browser and JavaScript environments with a focus on language interoperability
+and integration with existing JavaScript code.
```

Comment:
The flow here was weird. In the first paragraph we talk about JavaScript interop, but then at the end of the second paragraph we basically reintroduced the same idea ("and even JavaScript compilation"), so I just rearranged it.